packages/core/less/treegrid-table.less

```diff
@@ -0,0 +1,87 @@
+.chevron-position(@shift-left) {
+  left: -(@table-cell-padding*2.2 + @shift-left);
+}
+
+@show-less-icon: '\f105';
+@show-more-icon: '\f107';
+
+table {
+  &.tree-grid-table {
+    width: 100%;
+    max-width: 100%;
+  }
+
+  thead {
```

Comment:
These and following styles will affect all tables. I believe the styling here should only apply to `table.tree-grid-table`.